tailwind.config.ts

```diff
@@ -1,9 +1,34 @@
-export const content = [
-  './app/**/*.{js,ts,jsx,tsx,mdx}',
-  './pages/**/*.{js,ts,jsx,tsx,mdx}',
-  './components/**/*.{js,ts,jsx,tsx,mdx}',
-];
-export const theme = {
-  extend: {},
-};
-export const plugins = [];
+import type { Config } from 'tailwindcss';
+
+export default {
+  content: [
+    './app/**/*.{js,ts,jsx,tsx,mdx}',
+    './components/**/*.{js,ts,jsx,tsx,mdx}',
+  ],
+  theme: {
+    extend: {
+      // arbitrary name
+      colors: {
+        bgBlack: '2D2D2D',
+        bgGrey700: '7A7A7A',
+        bgGrey500: 'BCBABA',
+        bgGrey100: 'F9F9F9',
+        textBlack: '343434',
+        pointYellow: 'FF9900',
+      },
+      fontSize: {
+        h1: '32px',
+        h2: '20px',
+        p1: '16px',
+        p2: '14px',
+        p3: '12px',
+        p4: '10px',
+      },
+      fontWeight: {
+        bold: '700',
+        regular: '400',
+      },
```

Comment:
We need to discuss whether to declare these tokens in `extend` or directly declare them in the `theme`. If we declare directly in the `theme`, it will override the default Tailwind CSS classes, rendering some of Tailwind CSS's default properties inaccessible.

Because our design system is not yet fixed and robust, I declared them to `extend`. However, in this case, there is a possibility of confusion with the default Tailwind classes as well🥲